src/test/suite/fixtures/recorded/updateSelections/bringFineAfterThis2.yml

```diff
@@ -0,0 +1,32 @@
+spokenForm: bring fine after this
+languageId: plaintext
+command:
+  actionName: replaceWithTarget
+  partialTargets:
+    - type: primitive
+      mark: {type: decoratedSymbol, symbolColor: default, character: f}
+    - type: primitive
+      mark: {type: cursor}
+      position: after
+  extraArgs: []
+initialState:
+  documentContents: foo world
+  selections:
+    - anchor: {line: 0, character: 9}
+      active: {line: 0, character: 9}
+  marks:
+    default.f:
+      start: {line: 0, character: 0}
+      end: {line: 0, character: 3}
+finalState:
+  documentContents: foo world foo
+  selections:
+    - anchor: {line: 0, character: 9}
+      active: {line: 0, character: 9}
```

Comment:
I think this behaviour is kinda cool.  If we say "bring air after this", then the cursor doesn't move the way it usually does when you say "bring air"